### PR TITLE
Feature lpdev 566 invoices iata data

### PR DESCRIPTION
**Pull request by vitaliidasaev**
LPDEV-584 invoices ordered by CreatedDate

LPDEV-566 iata specific data

LPDEV-470 get filter for merchant

LPDEV-470 add hardcoded merchant logo to filter

### CODE REVIEW

File: src/Lykke.Service.PayAPI.Core/Services/IInvoiceService.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Lykke.Service.PayAPI.Core.Domain.Invoice;
+
+namespace Lykke.Service.PayAPI.Core.Services
+{
+    public interface IInvoiceService
```
Comment (NovichikhinAlexey) [inline src/Lykke.Service.PayAPI.Core/Services/IInvoiceService.cs]: IF i try understand this is client to external Iata service?
If it's try then rename this to IIataServiceClient

If not, please explane what is it

File: src/Lykke.Service.PayAPI/Controllers/Mobile/InvoicesController.cs [hunk 2]
```diff
@@ -158,11 +162,86 @@ public class InvoicesController : Controller
...
+                var groupMerchantsFilterItems = new List<MerchantFilterItemModel>();
+
+                foreach (var item in merchantsDictionary.ToListOfFilterItems())
+                {
+                    groupMerchantsFilterItems.Add(new MerchantFilterItemModel
+                    {
+                        Id = item.Id,
+                        Value = item.Value,
+                        //TODO: implement getting logo url later
+                        MerchantLogoUrl = "https://lkedevmerchant.blob.core.windows.net/merchantfiles/iata_256.jpg"
```
Comment (NovichikhinAlexey) [inline src/Lykke.Service.PayAPI/Controllers/Mobile/InvoicesController.cs]: Why hare HARDCODE?
move this to settings

File: src/Lykke.Service.PayAPI/Controllers/Mobile/InvoicesController.cs [hunk 3]
```diff
@@ -158,11 +162,86 @@ public class InvoicesController : Controller
...
+
+                var iataSpecificData = await _invoiceService.GetIataSpecificDataAsync(invoice.Id);
+                if (iataSpecificData != null)
+                {
+                    invoice.IataInvoiceDate = iataSpecificData.IataInvoiceDate;
+                    invoice.SettlementMonthPeriod = iataSpecificData.SettlementMonthPeriod;
+                }
+
+                //TODO: implement getting logo url later
+                invoice.LogoUrl = "https://lkedevmerchant.blob.core.windows.net/merchantfiles/iata_256.jpg";
```
Comment (NovichikhinAlexey) [inline src/Lykke.Service.PayAPI/Controllers/Mobile/InvoicesController.cs]: move to settings

File: src/Lykke.Service.PayAPI/Controllers/Mobile/InvoicesController.cs (hunk 2 above)
Comment (vitaliidasaev) [reply src/Lykke.Service.PayAPI/Controllers/Mobile/InvoicesController.cs]: it is temporary url for a few days while implementing logo tasks to put real urls, will move to Settings

File: src/Lykke.Service.PayAPI.Core/Services/IInvoiceService.cs (hunk 1 above)
Comment (vitaliidasaev) [reply src/Lykke.Service.PayAPI.Core/Services/IInvoiceService.cs]: Yes, it is for Iata data. The data are related to invoices that's why was such name ;). Will be renamed to IIataServiceClient

**Pull request merged**